compiler/rustc_session/src/config.rs

```diff
@@ -2182,6 +2182,13 @@ pub fn build_session_options(matches: &getopts::Matches) -> Options {
         }
     }
 
+    if !debugging_opts.unstable_options && cg.natvis.is_some() {
+        early_error(
+            error_format,
+            "`-C natvis` is unstable and must be used with the `-Z unstable-options` flag.",
```

Comment:
Can you please make it `-Z natvis` instead? `-Z` is where unstable options are normally placed. You can remove this stability check if you do this.

Reply:
In the RFC PR it was suggested to make this a `-C` flag gated behind the existing `-Z unstable-options` flag. This would allow for minimal changes in the path towards some future stabilization of this feature which is the long-term goal.